alf/layers.py

```diff
@@ -3131,6 +3154,8 @@ def __init__(self, module: Union[nn.Module, Callable], n: int):
         if isinstance(module, nn.Module):
             self._networks = nn.ModuleList(
                 [copy.deepcopy(module) for i in range(n)])
+            for net in self._networks:
+                reset_parameters(net)
```

Comment:
Was this a bug?

Reply:
Yes, this is a bug. It can be triggered if a module does not implement make_parallel but make_parallel_net is called.